src/Sulu/Bundle/SecurityBundle/Entity/User.php

```diff
@@ -25,14 +25,11 @@
 use Sulu\Component\Persistence\Model\AuditableTrait;
 use Sulu\Component\Security\Authentication\UserInterface;
 use Symfony\Component\Security\Core\User\EquatableInterface;
+use Symfony\Component\Security\Core\User\LegacyPasswordAuthenticatedUserInterface;
 use Symfony\Component\Security\Core\User\UserInterface as SymfonyUserInterface;
 
-/**
- * User.
- *
- * @ExclusionPolicy("all")
- */
-class User extends ApiEntity implements UserInterface, EquatableInterface, AuditableInterface, PasswordAuthenticatedUserInterface
+#[ExclusionPolicy('all')]
+class User extends ApiEntity implements UserInterface, EquatableInterface, AuditableInterface, PasswordAuthenticatedUserInterface, LegacyPasswordAuthenticatedUserInterface
```

Comment:
unexpected change. mostly this breaks the `lowest` version:

```suggestion
/**
 * @ExclusionPolicy("all")
 */
class User extends ApiEntity implements UserInterface, EquatableInterface, AuditableInterface, PasswordAuthenticatedUserInterface, LegacyPasswordAuthenticatedUserInterface
```